charts/redpanda/values.go

```diff
@@ -59,7 +59,7 @@ type Values struct {
 	AuditLogging     AuditLogging                  `json:"auditLogging"`
 	Enterprise       Enterprise                    `json:"enterprise"`
 	RackAwareness    RackAwareness                 `json:"rackAwareness"`
-	Console          Console                       `json:"console"`
+	Console          console.PartialValues         `json:"console"`
```

Comment:
Isn't this technically `console.Values` as helm injections console's `values.yaml` here?